src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -50,31 +52,16 @@ import {
   SkyFileItemService
 } from './file-item.service';
 
-// tslint:disable:no-forward-ref no-use-before-declare
-const SKY_FILE_ATTACHMENT_VALUE_ACCESSOR = {
-  provide: NG_VALUE_ACCESSOR,
-  useExisting: forwardRef(() => SkyFileAttachmentComponent),
-  multi: true
-};
-
-const SKY_FILE_ATTACHMENT_VALIDATOR = {
-  provide: NG_VALIDATORS,
-  useExisting: forwardRef(() => SkyFileAttachmentComponent),
-  multi: true
-};
-
```

Comment:
☝️ Implementing the `NgControl` injection pattern we used in the latest checkbox PR: #77